eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/NodeRelay.scala

```diff
@@ -54,12 +55,14 @@ object NodeRelay {
   sealed trait Command
   case class Relay(nodeRelayPacket: IncomingPaymentPacket.NodeRelayPacket) extends Command
   case object Stop extends Command
+  case object AsyncPaymentTrigger extends Command
```

Comment:
nit: I slightly prefer renaming it to something closer to the only other public command (`Relay`):
```suggestion
  case object RelayAsyncPayment extends Command
```

Reply:
Good idea; fixed in [3a632cd](https://github.com/ACINQ/eclair/pull/2435/commits/3a632cd6c127100ec9cb3fe9a207d80f325ebfea).